src/Amp/views/apache-vhost.php

```diff
@@ -5,12 +5,19 @@
  * @var string $host - the hostname to listen for
  * @var int $port - the port to listen for
  * @var string $include_vhost_file - the local path to a related config file
+ * @var string $visibility - which interfaces the vhost is available on
  */
 ?>
 
 <?php if ($use_listen) { ?>
 
+<?php if ($visibility === 'all'): ?>
 Listen <?php echo $port ?>
+<?php else: ?>
+Listen 127.0.0.1:<?php echo $port ?>
+
+Listen [::1]:<?php echo $port ?>
```

Comment:
At the risk of committing an Internet heresy... any idea how these different httpd's behave if the system doesn't have IPv6?